zerver/lib/test_classes.py

```diff
@@ -1313,6 +1314,26 @@ def check_user_subscribed_only_to_streams(self, user_name: str, streams: List[St
         for x, y in zip(subscribed_streams, streams):
             self.assertEqual(x["name"], y.name)
 
+    def mark_topic_resolved(
+        self,
+        user: UserProfile,
+        target_message_id: int,
+        **extra: str,
+    ) -> "TestHttpResponse":
+        """
+        Mark all messages within the topic associated with message `target_message_id` as resolved.
+        """
+        message, _ = access_message(user, target_message_id)
+        return self.api_patch(
+            user,
+            "/api/v1/messages/" + str(target_message_id),
```

Comment:
This could be improved using an f-string. Note a regression.

Reply:
Aha! You'd think with "implemented (a subset of) the original f-strings PEP" in my background I'd have defaulted to doing that refactor, but instead, I defaulted to "move the code as-is without modification".

Any chance there's been talk of a linter rule to gently nudge changes like this where they're appropriate? Something I found helpful in TypeScript land was ESLint complaining at me any time I failed to use template strings in a place where they made sense.